conf.d/downloads

```diff
@@ -6,7 +6,14 @@ dl() {
 }
 
 VERSION=$(gh_releases ladybirdweb/faveo-helpdesk | grep -v '^v\.' | sort -V | tail -1)
-URL="https://github.com/ladybirdweb/faveo-helpdesk/archive/${VERSION}.tar.gz"
+URL="https://github.com/ladybirdweb/faveo-helpdesk/archive/refs/tags/${VERSION}.tar.gz"
 SRC=/usr/local/src
 
 dl $URL $SRC
+
+
+#install inoncube loader
+systemctl restart apache2
+wget http://downloads3.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz
+tar -xvf ioncube_loaders_lin_x86-64.tar.gz
+cp ioncube/ioncube_loader_lin_8.2.so /usr/lib/php/20220829
```

Comment:
I'd move these 2 unpacking lines back the `conf.d/main` - with the ioncube config steps (sorry, perhaps I didn't explain myself well previously?). Although they'll need a tweak because we're now downloading it to `$SRC` (`/usr/local/src`). So in `conf.d/main`, I'd add:
```
tar xf $SRC/ioncube_loaders_lin_x86-64.tar.gz -C $SRC/
mv $SRC/ioncube/ioncube_loader_lin_8.2.so /usr/lib/php/20220829/
```
My next point is a relatively trivial one, especially for relatively small files where it likely makes little real world difference. Regardless, because these files will be cleaned up later, using `mv` here is better than `cp`. `cp` has to literally copy the data from one part of the disk to another part, whereas `mv` just moves the reference to the data on the disk (the data itself doesn't actually need to move). With large files the difference will be considerable (there are exceptions and details, but generally speaking).

Also, speaking of removing these downloaded files, I'm not seeing that happening anywhere (unless I'm missing it)? So you probably want a third line to go with the above 2:
```
rm -rf $SRC/ioncube*
```

Reply:
I moved the unpacking commands to the conf.d/main file. I also used mv instead of cp, and added the rm line.